src/components/common/ColonyActions/ColonyActions.tsx

```diff
@@ -1,33 +1,184 @@
-import { BigNumber } from 'ethers';
-import React from 'react';
-import { useColonyContext } from '~hooks';
+import React, { useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
 
-import { ActionTypes } from '~redux';
-import Button, { ActionButton } from '~shared/Button';
+import { SpinnerLoader } from '~shared/Preloaders';
+import LoadMoreButton from '~shared/LoadMoreButton';
+import { SortOptions } from '~shared/SortControls';
+import ActionsList from '~shared/ActionsList';
 
-const displayName = 'Common.ColonyActions';
+import { actionsSort, ActionsListHeading } from '.';
```

Comment:
I like this way of importing, however it's really bad for readability.

Don't need to do anything about it, just wanted to throw my 5 cents in :)